pkg/operator/validations.go

```diff
@@ -19,6 +19,17 @@ import (
 	"github.com/prometheus/common/model"
 )
 
+// ValidRelabelActions for relabelling
+var ValidRelabelActions = map[string]struct{}{
+	"replace":   {},
+	"keep":      {},
+	"drop":      {},
+	"hashmod":   {},
+	"labelmap":  {},
+	"labeldrop": {},
+	"labelkeep": {},
+}
```

Comment:
these are already defined here https://github.com/prometheus/prometheus/blob/4c56a193c518ae6f56008b0a4c850a9c3f1477c6/model/relabel/relabel.go#L44-L56 in a pkg we already import so might be worth considering reusing the consts?